source/MRViewer/MRRenderLabelObject.h

```diff
@@ -42,7 +52,16 @@ class RenderLabelObject : public IRenderObject
 
     // Marks dirty buffers that need to be uploaded to OpenGL
     mutable uint32_t dirty_;
+    mutable bool dirtySrc_;
     mutable bool dirtyBg_;
+    mutable bool dirtyLLine_;
+
+    struct LabelState
+    {
+        Box3f box;
+        Vector2f pivotShift;
+    };
+    mutable LabelState labelState_;
```

Comment:
i think pivot shift is enough for this propose

Reply:
Fixed.